applications/Web3Box.md

```diff
@@ -0,0 +1,137 @@
+
+2. Web3Box provides Polkadot users with risk assessment of parachains, so that Polkadot users can timely and accurately understand the risks of parachains; select the parachains with low risks for users to avoid those with high risks when making investment decisions.
+
+3. Web3Box presents the visual Web3 news for Polkadot users, including value-added content services and interactive narrative subjects, bringing better reading experience and more accurate content timeliness. Meanwhile, the unique aggregation function of Web3Box will be improved by Polkadot users in a decentralized manner, including Web3 industry sites, resources and tools. Finally, it will let Polkadot users liberate the conventional webpage favorites in Web2, achieving the one-stop access to all Web3 applications.
+
+
+## Team 
+
+### Team members
+
+- Name of team leader: Web3Box Founder Eric
+- Names of team members
+ Andrew 
+ More devs would be announced after passing the review
+
+### Contact
+
+-  Contact Name: Andrew 
+- Contact Email: andrew@web3box.cc
+- Website: https://web3box.cc
+
+### Legal Structure
+
+- We will update after passing the review
+
+### Team's experience
+
+Web3Box team is very experienced in software development and project operation. Team members are all aimed to be involved in Web3 Revolution.
+
+### Team Code Repos
+
+https://github.com/web3box-labs
+
+
+## Development Status
+
+Web3Box has confirmed the overall architecture. And now, Web3Box is confirming the feature development points based on the architecture. At the same time, frontend UI design work is undergoing.
+
+## Development Roadmap
+
+### Milestone 1 - Dashboard / Multi-chain Wallet
+
+- **Estimated duration:** 6 weeks
+- **FTE:**  2
+- **Costs:** 10,000 USD
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License |  MIT  |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can install and operate Web3Box client. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Article | We will publish an **article** in our official [Medium](https://medium.com/@web3box.official) that explains what was done/achieved as part of the grant.  |
+| 1. | **Dashboard**:Statistics and display of Polkadot users' assets | Statistics and display of the value of Polkadot users’ onchain assets, statistics and display of the value of input assets & output assets |
```

Comment:
Where do these prices come from?

Reply:
> Where do these prices come from?

For on-chain assets data, we will get this info from the supported chains' mainnet in real time, like Polkadot Relaychain, Acala Parachain, etc. And for the token price data source, we will get the data from [chainlink](https://chain.link/) every 5 mins or 10mins. All data sources are requested in a decentralized way.